lib/Minilla/Project.pm

```diff
@@ -616,7 +616,7 @@ sub generate_minil_toml {
     my $project_name = $self->_detect_project_name_from_dir;
     my $content      = join("\n",
         qq{name = "$project_name"},
-        qq{# badges = ["travis-ci.com"]},
+        qq{badges = ["github-actions/test"]},
```

Comment:
GitHub Actions would not need this comment out.

Reply:
github-actions/test -> github-actions/test.yml (cf #317)

Reply:
Thanks! fixed it.
https://github.com/tokuhirom/Minilla/pull/318/commits/93ae40794aaf85d7dc34d1b123d12c37f6fcc140